qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -31,8 +31,9 @@ set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOUR
 set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER})
 
+ament_get_recursive_properties(deps_include_dirs _ ${pluginlib_TARGETS})
+
 set(_qt_gui_cpp_sip_LIBRARIES
-  ${pluginlib_LIBRARIES}
```

Comment:
I guess we should continue to pass pluginlib dependencies here?

Reply:
Readded in 6056395de9deba2bac92b733023a41358c439829.